cmake/wxWidgets/CMakeLists.txt

```diff
@@ -200,3 +251,36 @@ if(NOT PCHSupport_FOUND)
 endif()
 
 ADD_PRECOMPILED_HEADER(${PROJECT_NAME} ${SOURCE_DIR}/wxWidgets/include/wx/wxprec.h)
+
+set(WEBVIEW2_VERSION "1.0.705.50")
+set(WEBVIEW2_URL "https://www.nuget.org/api/v2/package/Microsoft.Web.WebView2/${WEBVIEW2_VERSION}")
+set(WEBVIEW2_SHA256 "6a34bb553e18cfac7297b4031f3eac2558e439f8d16a45945c22945ac404105d")
```

Comment:
This is going to be a PITA to maintain... I'd much prefer if it used the nuget tool or at least information from `packages.config`.

Reply:
Added some simple regex-based parsing logic to grab the version from `packages.config`.